src/components/CommentBox.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react"
+import { getComments } from "../api"
+import CommentCard from "./CommentCard"
+
+export default function CommentBox (props){
+    const [comments, setComments] = useState([])
+    const { articleId } = props
+    
+    useEffect(() => {
+        console.log("getting comments")
+        getComments(articleId).then((comments) => {
+            setComments(comments)
+        }).catch((err) => {
+            console.log("🚀 ~ getComments ~ err:", err)
+        })
+    },[])
+
+    return (
+        <ul>
+        {
+            comments.map((comment) => {
```

Comment:
Great job Zack! Another scenario we need to cover, is if there are no comments yet on an article. Rather than leave the area blank, we want to ideally give the user feeback on this and rendering something like 'no comments yet'